molo/profiles/views.py

```diff
@@ -223,14 +223,19 @@ def form_valid(self, form):
         # check security question answers
         answer_checks = []
         for i in range(profile_settings.num_security_questions):
-                user_answer = form.cleaned_data["question_%s" % (i,)]
+            user_answer = form.cleaned_data["question_%s" % (i,)]
+            try:
                 saved_answer = user.profile.securityanswer_set.get(
                     user=user.profile,
                     question=self.security_questions[i]
                 )
-                answer_checks.append(
-                    saved_answer.check_answer(user_answer)
-                )
+                answer_checks.append(saved_answer.check_answer(user_answer))
+            except SecurityAnswer.DoesNotExist:
```

Comment:
We should perhaps be more specific and say you have not answered any security questions?

Reply:
Also, if we do this, we should give them the option to answer the questions if they have not done so already